### ISSUE TITLE
handling code validation

### ISSUE THREAD

**Issue by wanderer** [ewasm/design#59]
On account creation the if the code is invalid should it OOG?

**Comment by lrettig** [comment]
Looked for the EIP and couldn't find it, I'd be curious to see it.

**Comment by axic** [comment]
Agreed to codify what is currently implemented: contract verification failure results in OOG. Need to document this.
